build/release/README.md

```diff
@@ -1,34 +1,69 @@
-# lnd's Reproducible Build System
+# `lnd`'s Reproducible Build System
 
-This package contains the buidl script that the `lnd` project uses in order to
-build binaries for each new release. As of Go 1.13, with some new build flags,
+This package contains the build script that the `lnd` project uses in order to
```

Comment:
kek

good news, i grepped and this is the _only_ occurrence of buidl in the codebase